ts/private/ts_project.bzl

```diff
@@ -153,22 +154,28 @@ This might be because
 - `srcs` has elements producing non-ts outputs
 """ % label
         else:
-            no_outs_msg = "ts_project target %s with custom transpiler needs `declaration = True`." % label
+            no_outs_msg = "ts_project target %s with custom transpiler needs 'declaration = True'." % label
```

Comment:
Should this not use backticks?

Reply:
This is intentional. Backticks for anything that goes into docs and single quotes for CLI messages.